pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -51,11 +53,23 @@ func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfigura
 		glog.V(3).Infof("Problem accessing %s: %v", dockercfgPath, err)
 		return docker.AuthConfiguration{}, false
 	}
-	cfg, err := readDockercfg(dockercfgPath)
-	if err != nil {
-		glog.Errorf("Reading %s failed: %v", dockercfgPath, err)
-		return docker.AuthConfiguration{}, false
+
```

Comment:
Before line 52 it might be helpful to add a check for dockercfgPath  == "" and just say we couldn't find a path to the config. Otherwise the os.Stat error is not as helpful
```
stat : no such file or directory
```